cmd/kubetoken/kubetoken.go

```diff
@@ -54,25 +54,36 @@ You can find more information on 'ExecCredential' and 'client side authenticatio
 	cmd.RunE = func(_ *cobra.Command, args []string) error {
 		ctx := context.Background()
 
-		err := contextCMD.NewContextCommand().Run(ctx, &contextCMD.ContextOptions{
+		k8sClientProvider := okteto.NewK8sClientProvider()
+		okClientProvider := okteto.NewOktetoClientProvider()
+		err := newPreReqValidator(
+			withCtxName(contextName),
+			withNamespace(namespace),
+			withK8sClientProvider(k8sClientProvider),
+			withOktetoClientProvider(okClientProvider),
+		).Validate(ctx)
+		if err != nil {
+			return fmt.Errorf("failed to validate pre-reqs: %w", err)
+		}
+
+		err = contextCMD.NewContextCommand().Run(ctx, &contextCMD.ContextOptions{
 			Context:   contextName,
 			Namespace: namespace,
 		})
 		if err != nil {
 			return err
 		}
```

Comment:
```suggestion
		if err := contextCMD.NewContextCommand().Run(ctx, &contextCMD.ContextOptions{
			Context:   contextName,
			Namespace: namespace,
		}); err != nil {
			return err
		}
```

Reply:
It's usually a smell code to have a large if because it reduces visibility, that's why i prefer to have it separate. You can read more about  this [here](https://blog.jetbrains.com/idea/2017/09/code-smells-if-statements/)